src/org/omegat/core/statistics/FindMatches.java

```diff
@@ -275,7 +302,7 @@ public List<NearString> search(String searchText, boolean requiresTranslation, b
                 for (String onesrc : segments) {
                     // find match for a separate segment
```

Comment:
```suggestion
                    // find match for a separate segment.
                    // WARN: the 5th argument should be
                    // `false` to avoid an infinite-loop.
```

it is better to give warning.